src/auto-imports.d.ts

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable */
```

Comment:
is this generated from somewhere? You can add it to .gitattributes to mark it as such for nicer diffs https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

```
src/*.d.ts linguist-generated=true
```

but it does make me wonder how it is generated and if I'd need to do it again?